detector/zoo/plugin_test.go

```diff
@@ -0,0 +1,19 @@
+package zoo
```

Comment:
Good deal!! Moving the test in zoo makes more sense then what I was trying to do.